internal/configure/pool.go

```diff
@@ -96,7 +97,7 @@ type (
  *   logicalpools:
  *     - name: pool1
  *       physicalpool: pool1
- *       replicasnum: 3
+ *       instancessum: 3
```

Comment:
Don't change it, the `replicasnum` represents the number of copies of the data, not the number of services, this is why we want to need this PR :)